.github/workflows/publish-snapshot.yml

```diff
@@ -9,19 +9,13 @@ jobs:
     steps:
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4
 
-      - name: Cache Gradle Files
-        uses: actions/cache@13aacd865c20de90d75de3b17ebe84f7a17d57d2 # v4
-        with:
-          path: |
-            ~/.gradle/caches/
-            ~/.gradle/wrapper/
-          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
-          restore-keys: |
-            ${{ runner.os }}-gradle-
+      - name: Setup gradle
+        uses: gradle/actions/setup-gradle@v3
+
       - name: Set up Java
         uses: actions/setup-java@387ac29b308b003ca37ba93a6cab5eb57c8f5f93 # v4
         with:
-          distribution: 'zulu'
+          distribution: 'temurin'
```

Comment:
```suggestion
          distribution: 'zulu'
```

Reply:
Strong no! Temurin is pre-installed:
https://github.com/TWiStErRob/github-actions-test/actions/runs/7746795967/job/21125817607#step:3:1

Reply:
ah, ok, re  https://github.com/detekt/sarif4k/pull/81#discussion_r1474847372, will raise another PR

Reply:
https://github.com/detekt/sarif4k/pull/88